blob/gcsblob/gcsblob.go

```diff
@@ -122,9 +122,12 @@ type Options struct {
 }
 
 // openBucket returns a GCS Bucket that communicates using the given HTTP client.
-func openBucket(ctx context.Context, bucketName string, client *gcp.HTTPClient, opts *Options) (driver.Bucket, error) {
+func openBucket(ctx context.Context, bucketName string, client *gcp.HTTPClient, opts *Options) (*bucket, error) {
 	if client == nil {
-		return nil, errors.New("OpenBucket requires an HTTP client")
+		return nil, errors.New("gcsblob.OpenBucket: client is required")
+	}
+	if bucketName == "" {
```

Comment:
We used to have a simple check for bucket and object names, we then decided to remove it and rely on the backend service to check for the name. Is there any reason to add an empty name check now?